### DISCUSSION TITLE
Still Requiring Purchase After Paying

### DISCUSSION THREAD

**Discussion by SpencerWilhelmy** [noah-nuebling/mac-mouse-fix#1021] [Questions] [answered]
I recently purchased the new $2.99 requirement for using Mac Mouse Fix but for some reason it had me download the app again and now it doesn't recognize that I have purchased. What do I do?

**Comment by noah-nuebling** [comment, marked as answer]
Hey @SpencerWilhelmy, thanks for your feedback! 

Have you tried entering your license key?

<img width="280" alt="Screenshot 2024-06-26 at 18 45 16" src="https://github.com/noah-nuebling/mac-mouse-fix/assets/40808343/871140af-3fe9-4865-ab6e-43df1501dbc5">

You don't have to download the app again. But this is very valuable feedback so I can improve the user experience. I assume you clicked on the "Download" button on the screen that appears right after you purchased the app on Gumroad.com? (On that screen you could also see your license key)

**Reply by SpencerWilhelmy** [reply]
Hey @noah-nuebling, where would I re-find that key? I think that is what happened.

**Reply by noah-nuebling** [reply]
Hey Spencer, thanks for the feedback! You should be able to find the license key in your email inbox.

**Reply by SpencerWilhelmy** [reply]
@noah-nuebling Ah of course, duh. Thanks it works now :) and thanks for making it! Makes my $70 mouse not useless

**Reply by noah-nuebling** [reply]
Thanks for buying the app, enjoy!
